content/docs/k8s/gateway-api.mdx

```diff
@@ -2,15 +2,17 @@
 title: Gateway API
 sidebar_position: 3
 lang: en-US
-keywords: [kubernetes, k8s, gateway api]
+keywords: [kubernetes, k8s, gateway api, ingress, ingress controller]
 ---
 
-Starting in v0.28, the Pomerium Ingress Controller includes experiment support for the Kubernetes [Gateway API].
```

Comment:
I think I have added a "new in v0.XX" in a few other places in the past, but I'll leave this out if you prefer.

I'm not seeing links to the ingress-controller repo in the other pages in this section, so I'd prefer to leave this unlinked for consistency.